new-frontiers-14/frontier-station-14
# Adds Medical Assembler to FSB Stasis

## About the PR



Adds:
 - Medical assembler in chem lab (Thanks Whatstone!)
 - Max pump to waste system (Thanks Whatstone again)
 - Button in cockpit to control thruster power (Thanks Darin!)
 - Button in medical ward to lock doors (Thanks upstream?)
 - Chemistry wall shelf near cryotubes (Thanks feedback!)

Tweaks:
- Slight change to structure near airlock. Better configuration I feel.
- Changes location of ATM and gas dispenser to/in airlock/staging area.
- EMS signs/posters replacing old medical signs.
- Erhard new wall lockers with fills replacing old wall lockers (Thanks Erhard!)
- Moved cryotube defib over by one.

Removed:
- Scarves 

## Why / Balance


New medical assembler and visuals on wall lockers are good.
New controls over thrusters and doors are good.
Making sure my stuff is up to date and maintained. As one does.

## How to test


Check lockers and fills.
Check buttons. New in cockpit and new by stasis beds.
Ensure waste is pumping out (new pump).

## Media


![ss+(2024-10-12+at+10 01 38)](https://github.com/user-attachments/assets/0bc61f89-5949-4842-aa34-f657ec63cc36)

## Requirements



## Breaking changes


None
**Changelog**


🆑 
- Add: FSB Stasis now boasts a medical assembler.
